### DISCUSSION TITLE
Idle inhibitor

### DISCUSSION THREAD

**Discussion by mahaveergurjar** [prasanthrangan/hyprdots#417] [Q&A]
Is any option or method to enable idle inhibitor automatically enable when video is running on youtube or locally in system like this feature is in kde already available

**Comment by kRHYME7** [comment]
Hi you can use sway-audio-idle-inhibit .  Check their GitHub for instructions. I forgot but I think I answered a similar query about this.

**Comment by mahaveergurjar** [comment]
Ok Thanks

**Comment by kRHYME7** [comment]
Hi @mahaveergurjar  here is the discussion also you can add this to your ./userprefs.conf #303
